scripts/mbedtls_framework/test_case.py

```diff
@@ -89,3 +115,25 @@ def write_data_file(filename: str,
             tc.write(out)
         out.write('\n# End of automatically generated file.\n')
     os.replace(tempfile, filename)
+
+def psa_or_3_6_feature_macro(psa_name: str,
+                             domain_3_6: Domain36) -> str:
+    """Determine the dependency symbol for a given psa_alg based on
```

Comment:
Minor: should rename to `psa_name`. Also, please clarify that the input is the name for the PSA mechanism itself, (ie `PSA_ALG`, `PSA_KEY_TYPE` etc), not the `PSA_WANT` macro.